src/dashboard/category-results-screen.css

```diff
@@ -81,4 +81,41 @@
   top: 50%;
   margin-top: -0.25em;
   margin-left: -1.125em;
-}
\ No newline at end of file
+}
+
+.tooltip {
```

Comment:
I wouldn't recommend burying info in tooltips. Also this gets more complicated in emails, mobile etc.